packages/wallet-adapter-core/src/ans.ts

```diff
@@ -0,0 +1,21 @@
+const ChainIdToAnsContractAddressMap: Record<string, string> = {
+  "1": "mainnet", // mainnet
+  "2": "testnet", // testnet
```

Comment:
I didn't realize testnet worked with ANS still, probably needs an update in the explorer too